components/board/odroid/board.go

```diff
@@ -0,0 +1,27 @@
+// Package odroid implements a odroid based board.
+package odroid
+
+import (
+	"errors"
+
+	"periph.io/x/host/v3"
+
+	"go.viam.com/rdk/components/board/genericlinux"
+	"go.viam.com/rdk/logging"
+)
+
+const modelName = "odroid"
```

Comment:
Discussion question: is this the right model name? Odroid makes a dozen boards, but right now we only support one. I see two paths forward:
1. Make the model name "odroid" and in the future add multiple entries to the `boardInfoMappings` to support multiple boards.
2. Make the model name "odroid-c4" and in the future create new components for "odroid-n2," "odroid-m1," "odroid-h3," etc.

Approach 1 is useful because we don't overwhelm non-Odroid users with a zillion options. Approach 2 is useful because we don't mislead people into thinking we support boards that we don't actually support. What's the right one? I'm unsure...

Reply:
...no, what you've got already is right. It matches what we did with "jetson" and "beaglebone" and frankly "pi": the model is an entire brand name, and we only support a small number of boards from that brand. If we're going to change it for one, we should change it for all. Keep as-is for consistency.